h2o-core/src/main/java/water/H2O.java

```diff
@@ -1986,6 +1999,25 @@ public static void main( String[] args ) {
     long time5 = System.currentTimeMillis();
     startLocalNode();
 
+    if (ARGS.notify != null && !ARGS.notify.trim().isEmpty()) {
```

Comment:
Not sure if this is the best place where to put it. We also have a method "H2O.notifyAboutCloudSize" that we could use for this purpose. It would probably be more consistent.